ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/WorkloadCvesDeploymentSinglePage.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+function WorkloadCvesDeploymentSinglePage() {
+    const { deploymentId } = useParams();
+    return <>Workload CVE Deployment Single Page: {deploymentId}</>;
```

Comment:
Do you want to render `PageTitle` elements in this pull request, or as a follow up?

Reply:
I'd say as a follow up. I've got Jira tickets created that address the main layout of the page (breadcrumbs, etc). This change was primarily to get the routing in place, and the plain text was a way to verify that things were working correctly in the UI.